calico-enterprise_versioned_docs/version-3.16/network-policy/application-layer-policies/alp-tutorial.mdx

```diff
@@ -5,12 +5,30 @@ description: Learn how to apply ALP to your workloads and control ingress traffi
 # Application layer policy tutorial
 
 This tutorial shows how to use {{prodname}} application layer policy to restrict ingress traffic for applications and microservices.
-Assume that the application is installed in your {{prodname}} cluster and that the following application pods are running. 
+ 
+### Install the demo application
 
-## Set up
-- A {{prodname}} cluster is running with application layer policy enabled
-- Cluster has three microservices: customer, database, summary
-- The customer web service should not have access to the backend database, but should have access to clients outside the cluster
+We will use a simple microservice application to demonstrate {{prodname}}
+application layer policy. The [YAO Bank](https://github.com/projectcalico/yaobank) application creates a
+customer-facing web application, a microservice that serves up account
+summaries, and an [etcd](https://github.com/coreos/etcd) datastore.
+
+```bash
+kubectl apply -f {{tutorialFilesURL}}/10-yaobank.yaml
+```
+
+:::note
+
+You can also
+[view the manifest in your browser](/files/10-yaobank.yaml).
+
+:::
+
+Verify that the application pods have been created and are ready.
+
+    kubectl get pods
```

Comment:
```suggestion
    bash
    kubectl get pods
```

Reply:
Fixed

Reply:
[minor thing] this can be left as is, but let me share you my preference on verifying application pods: 

rather than displaying all the pods and hunting-pecking for the pods of interest, I prefer explicitly checking `rollout status` of specified deployments instead. 

For example, you are applying the  `yaobank` manifest:

```bash
➜  kubectl apply -f ./yaobank/10-yaobank.yaml 
namespace/yaobank configured
service/database created
serviceaccount/database created
deployment.apps/database created
service/summary created
serviceaccount/summary created
deployment.apps/summary created
service/customer created
serviceaccount/customer created
deployment.apps/customer created
```

Then you can immediately check the status of your deployments contained on that manifest
```bash
➜  kubectl rollout status -f ./yaobank/10-yaobank.yaml
Waiting for deployment "database" rollout to finish: 0 of 1 updated replicas are available...
deployment "database" successfully rolled out
Waiting for deployment "summary" rollout to finish: 0 of 1 updated replicas are available...
deployment "summary" successfully rolled out
deployment "customer" successfully rolled out
no status viewer has been implemented for Namespace
no status viewer has been implemented for Service
no status viewer has been implemented for ServiceAccount
no status viewer has been implemented for Service
no status viewer has been implemented for ServiceAccount
no status viewer has been implemented for Service
no status viewer has been implemented for ServiceAccount
```

NB: the 'no status viewer implemented for' warnings are safe to ignore. 


Alternatively, you can explicitly check the rollout status of deployments you care about
```bash
kubectl rollout status deploy/summary deploy/customer deploy/database
```